### DISCUSSION TITLE
Maximum number of listing / variation pictures?

### DISCUSSION THREAD

**Discussion by geolectric** [etsy/open-api#763] [Q&A] [answered]
I'm hoping someone can clarify this for me... I'm looking into using variations and having pictures for each variation type.

Is there a maximum of 10 listing pictures total - including variation pictures?

Or can you add the 10 standard listing pictures, and then additional pictures for each variation?

How many maximum pictures can you have for a listing - including variations?

**Comment by etsyachristensen** [comment, marked as answer]
Your listing can have up to 10 images total. You can then assign any of those 10 photos to your variations.

**Reply by geolectric** [reply]
@etsyachristensen  Thank you, and from using the Etsy UI - it appears each variation type can only be attached to 1 of the listing pictures?

For example, if I used the "Style" variation and added "Canvas" and "Print" as the two different types. Then I can only assign 1 of the 10 pictures to "Canvas" and 1 to "Print"?

**Reply by etsyachristensen** [reply]
That is correct @dropdx . Each variant option can have one photo.
